my_first_bioinf_package.py

```diff
@@ -0,0 +1,69 @@
+from inner_code import filter_gc_content, filter_length, filter_quality
+from typing import Tuple, Union
+def filter_fastq (seqs: dict, length_bounds: Union [tuple, int] =(0, 2**32), gc_bounds: Union [tuple, int] =(0, 100), quality_threshold: int=0) -> dict: 

```

Comment:
Тут несколько важных моментов.
1. импорт из собственной библиотеки - inner_code в данном случае является библиотекой, из которой ты хочешь испортировать функции своих скриптов. В самой папке содержатся только имена скриптов, но не функций, поэтому такой импорт не сработает (этт комментарий относится ко всем импортам в данном коде)
2. Порядок импортов - сначала идут стандартные библиотеки, потом внешние, потом собственные 
3. Пустые строки согласно PEP8 после импортов
4. Здорово, что ты решила импортировать сразу функции - зачастую, особенно, если модуль написан тобой и как следствие, имена функций уникальные, их так удобнее использовать.
```suggestion
from typing import Tuple, Union

from inner_code.filter_functions import filter_gc_content, filter_length, filter_quality
from inner_code.dna_functions import count_aa_length, count_nucl_length
from inner_code.protein_functions import is_dna, check_ecor1_site, count_gc_content, count_dna_melting, reverse, transcribe, complement, reverse_complement


def filter_fastq(seqs: dict, length_bounds: Union [tuple, int] =(0, 2**32), gc_bounds: Union [tuple, int] =(0, 100), quality_threshold: int=0) -> dict: 

```